src/main/webapp/app/quiz/participate/multiple-choice-question/multiple-choice-question.component.html

```diff
@@ -24,6 +25,9 @@ <h4 class="question-title-display"><span>{{questionIndex}})</span> {{question.ti
     <div class="question-score result" *ngIf="showResult && !forceSampleSolution" [ngClass]="{incorrect: (score || 0) < question.score}">
         <span jhiTranslate="arTeMiSApp.question.yourScore" class="colon-suffix"></span>
         <span>{{score || 0}}/{{question.score}}</span>
+        <jhi-quiz-scoring-infostudent-modal [score]="score" [question]="question" [correctAnswer]="correctAnswers"
+                                            [correctlyChosenAnswers]="chosenCorrectAnswerOption"
+                                            [wronglyChosenAnswers]="chosenWrongAnswerOption" [amountOfAnswerOptions]="amountOfAnswerOptions" ></jhi-quiz-scoring-infostudent-modal>
     </div>
```

Comment:
You pass a lot of variables here. I think it would be better, if you only pass the question variable and the submittedAnswer variable. You should be able to calculate all others values inside the quiz-scoring-info component